examples/python/addition.py

```diff
@@ -0,0 +1,19 @@
+import os
+import tensorflow as tf
+
+# Turn off TensorFlow warning messages in program output
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '2'
+
+# Define computational graph
+X = tf.placeholder(tf.float32, name="X")
+Y = tf.placeholder(tf.float32, name="Y")
+
+addition = tf.add(X, Y, name="addition")
+
+
+# Create the session
```

Comment:
Just to follow python's best practices we should consider having a `requirements.txt` and using a `main` function: 

```python
import os
import tensorflow as tf

os.environ['TF_CPP_MIN_LOG_LEVEL'] = '2'

def main(): 
    X = tf.placeholder(tf.float32, name="X")
    Y = tf.placeholder(tf.float32, name="Y")
    addition = tf.add(X, Y, name="addition")

    with tf.Session() as session:
        result = session.run(addition, feed_dict={X: [1, 2, 10], Y: [4, 2, 10]})
        print(result)

if __name__ == '__main__':
  main()
``` 

With your comments, of course (: